src/pages/in/invoice-software.js

```
@@ -1,9 +1,1158 @@
+      </section>
+
+      <section className='container-fluid features__description_container mt-5'>
+        <div className='container'>
+          {isGlobal ? (
+            <>
+              {globalData.map((feature, index) => {
+                return (
+                  <div
+                    className={`row mb-5 pb-5 align-items-center border-bottom features__description_container--row-${index % 2 === 0 ? "even" : "odd"
+                      }`}
+                  >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)